worker/draft.go

```diff
@@ -606,7 +623,7 @@ func (n *node) applyMutations(ctx context.Context, proposal *pb.Proposal) (rerr
 		}
 
 		for _, tupdate := range proposal.Mutations.Types {
-			if err := runTypeMutation(ctx, tupdate); err != nil {
+			if err := runTypeMutation(ctx, tupdate, startTs); err != nil {
```

Comment:
The startTs is actually the `MaxAssigned`. See line 605. You might end up writing data at the wrong timestamp.